pom.xml

```diff
@@ -8583,6 +8584,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.resteasy</groupId>
+                <artifactId>resteasy-tracing-api</artifactId>
+                <version>${version.org.jboss.resteasy.tracing.api}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
There needs to be an entry in the https://github.com/wildfly/wildfly/blob/main/ee-feature-pack/common/src/main/resources/license/ee-feature-pack-common-licenses.xml file for the licenses.

Reply:
okay!